commcare_connect/templates/tailwind/components/upload_progress_bar.html

```diff
@@ -14,25 +14,79 @@
     </div>
 
   {% elif not progress.complete %}
-  <div>
-    <div class="w-full h-2 bg-gray-200 rounded-full overflow-hidden mb-4">
-      <div class="h-full bg-purple-600 rounded-full animate-[progress_2s_linear_infinite]"
-           style="width: 80%; background-image: linear-gradient(to right, #8b5cf6, #a78bfa, #8b5cf6); background-size: 200% 100%;"></div>
+    <div class="w-full">
+      <div class="w-full h-2 bg-gray-200 rounded-full overflow-hidden mb-4">
+        <div
+          class="h-full bg-purple-600 rounded-full animate-[progress_2s_linear_infinite]"
+          style="width: 80%; background-image: linear-gradient(to right, #8b5cf6, #a78bfa, #8b5cf6); background-size: 200% 100%;">
+        </div>
+      </div>
+      <p class="w-full text-sm text-gray-700">
+        {% if progress.message %}
```

Comment:
I think this change got missed in tailwind migration. Originally added [here](https://github.com/dimagi/commcare-connect/pull/492/files#diff-5de9a1b7e069a39ac2833ba3a23b329db52860d819b9e25901900bc7c4caeb74)